.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thanks for contributing! -->
+[Issue Key](https://jira.mongodb.org/browse/%7BISSUE_KEY%7D)
+## Summary
+<!-- What conceptually is this PR introducing? If context is already provided from the JIRA ticket, still place it in the
+Pull Request as you should not make the reviewer do digging for a basic summary. -->
+
+## Changes in this PR
+<!-- What changes did you make to the code? What new APIs (public or private) were added, removed, or edited to generate
+the desired outcome explained in the above summary? -->
+
+## Testing Plan
+<!-- How did you test the code? If you added unit tests, you can say that. If you didn’t introduce unit tests, explain why.
+All code should be tested in some way – so please list what your validation strategy was. -->
+
+### Screenshots (optional)
+<!-- Usually a great supplement to a test plan, especially if this requires local testing. -->
+
+## Checklist
+<!-- Do not delete the items provided on this checklist. -->
+### Checklist for Author
+- [ ] Did you update the changelog (if necessary)?
+- [ ] Is the intention of the code captured in relevant tests?
+- [ ] If there are new TODOs, has a related JIRA ticket been created?
+
+### Checklist for Reviewer {@primary_reviewer}
+- [ ] Does the title of the PR reference a JIRA Ticket?
+- [ ] Do you fully understand the implementation? (Would you be comfortable explaining how this code works to someone else?)
+- [ ] Have you checked for spelling & grammar errors?
+- [ ] Is all relevant documentation (README or docstring) updated?
+
+## Focus Areas for Reviewer
```

Comment:
Is this optional like Screenshots?

Reply:
The template that we discussed didn't explicitly mark it as optional and I'd imagine this would be used more often than screenshots? But i also can see situations where there aren't any focus areas to highlight? So not entirely sure. 
What does everyone else think? (@Jibola @caseyclements )